### PR TITLE
Make container size more configurable

### PR DESCRIPTION
**Pull request by krissi**
This PR adds more control over the storage size while trying to communicate how the storage is calculated.

### CODE REVIEW

File: lib/lib_storagetank.lua
Comment (nicolas-lang) [inline lib/lib_storagetank.lua:1]: I think I remember that the max size had some kind of perforemance reason. I would prefer to keep this.

**Review by nicolas-lang — CHANGES_REQUESTED** [review]
Except the removal of the max size I like your changes.
Can you please review my comment, in the meantime I'll have a look on the release process, it seems like the release pipeline is broken :-(

File: lib/lib_storagetank.lua
Comment (krissi) [reply lib/lib_storagetank.lua:1]: I readded the size limitation and like in the warehouse mod I added a startup setting to disable the limit if desired

**Comment by nicolas-lang** [comment]
not sure if I'll have time to release this weekend, I'll try to fix the release pipeline later this evening

**Comment by nicolas-lang** [comment]
released.

**Pull request merged**